tools/templates/vfvs_run.py

```diff
@@ -315,11 +606,199 @@ def process_ligand(task):
+                logging.error(
+                    f"Could not find score for {task['collection_key']} {task['ligand_key']} {task['scenario_key']} {task['replica_index']}")
+            # Delete/move auxillary files: 
+            os.system('rm min.xyz')
+            os.system('cp out.xyz {}'.format(task['output_path']))
+            os.system('rm out.xyz conformers.xyz')
+            
+        elif(task['program'] == "LigandFit"): 
+
+            try: 
+                with open('./LigandFit_run_1_/ligand_1_1.log', 'r') as f: 
+                    lines = f.readlines()
+                lines = [x for x in lines if 'Best score' in x]
+                scores = []
+                for item in lines: 
+                    scores.append( float([x for x in item.split(' ') if x != ''][-2]) )
+                completion_event['score'] = min(scores)
+                completion_event['status'] = "success"
+            except: 
+                logging.error(
+                    f"Could not find score for {task['collection_key']} {task['ligand_key']} {task['scenario_key']} {task['replica_index']}")
+            os.system('rm -rf LigandFit_run_1_')
+        
+        elif(task['program'] == "ledock"): 
+            try: 
+                os.system('cp ./ligands/{}.dok {}'.format(task['ligand_path'].split('.')[0], task['output_path']))
+                with open('{}.dok'.format(task['ligand_path'].split('.')[0]), 'r') as f: 
+                    lines = f.readlines()
+                lines = [x for x in lines if 'Score' in x]
+                scores = []
+                for item in lines: 
+                    A = item.split('Score')[-1].strip().split(': ')[1].split(' ')[0]
+                    scores.append(float(A))
+                completion_event['score'] = min(scores)
+                completion_event['status'] = "success"
+            except: 
+                logging.error(
+                    f"Could not find score for {task['collection_key']} {task['ligand_key']} {task['scenario_key']} {task['replica_index']}")
+            os.system('rm {} {}'.format(cmd[-1], cmd[-1].split('.')[0]+'.list'))
+        
+        elif(task['program'] == "gold"): 
+            
+            try: 
+                os.system('cp {}/gold_ligand_m1.mol2 {}'.format(cmd[-1].split('.')[0], task['output_path']))
+                with open('./{}/ligand_m1.rnk'.format(cmd[-1].split('.')[0]), 'r') as f: 
+                    lines = f.readlines()
+                docking_score = float([x for x in lines[-1].split(' ') if x!=''][1])
+                completion_event['score'] = min(docking_score)
+                completion_event['status'] = "success"
+            except: 
+                logging.error(
+                    f"Could not find score for {task['collection_key']} {task['ligand_key']} {task['scenario_key']} {task['replica_index']}")
+            os.system('rm -rf {} {}'.format(cmd[-1].split('.')[0], cmd[-1]))
+
+        elif(task['program'] == "gold"): 
```

Comment:
gold is listed twice

Reply:
THis is a bug. Will update (this is supposed to be another docking software)